src/main/scala/com/gu/datalakealerts/WorkerLambda.scala

```diff
@@ -114,14 +112,28 @@ object WorkerLambda {
     }
   }
 
-}
-
-object TestWorker {
-  def main(args: Array[String]): Unit = {
-    if (args.length != 2) {
-      logger.error("Please provide a platformId and a featureId e.g. sbt \"runMain com.gu.datalakealerts.TestWorker android friction_screen\"")
+  def process(env: WorkerEnv, feature: Feature, platform: Platform, queryExecutionId: String): Unit = {
+    val queryStatus = Athena.retrieveQueryStatus(queryExecutionId)
+    if (Athena.queryHasSuccessfulState(queryStatus)) {
+      analyseQueryResults(feature, platform, queryExecutionId)
+    } else if (Athena.queryHasUnsuccessfulState(queryStatus)) {
```

Comment:
This is new behaviour - I don't think we'd have handled this very gracefully before!